azurerm/internal/services/web/resource_arm_function_app.go

```diff
@@ -790,6 +877,32 @@ func flattenFunctionAppSiteConfig(input *web.SiteConfig) []interface{} {
 		result["http2_enabled"] = *input.HTTP20Enabled
 	}
 
+	restrictions := make([]interface{}, 0)
+	if vs := input.IPSecurityRestrictions; vs != nil {
+		for _, v := range *vs {
+			block := make(map[string]interface{})
+			if ip := v.IPAddress; ip != nil {
+				// the 2018-02-01 API uses CIDR format (a.b.c.d/x), so translate that back to IP and mask
+				if strings.Contains(*ip, "/") {
+					ipAddr, ipNet, _ := net.ParseCIDR(*ip)
+					block["ip_address"] = ipAddr.String()
+					mask := net.IP(ipNet.Mask)
+					block["subnet_mask"] = mask.String()
+				} else {
+					block["ip_address"] = *ip
+				}
```

Comment:
Could we make the behave like the new API expects, no subnet mask and ip address in cidr format

Reply:
Fixed